super_team_builder/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "super_team_builder",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js"
```

Comment:
Where possible, try to include scripts for creating and deleting dbs at minimum, so any users setting up your project have an easier time. eg:
```
"scripts": {
    "start": "DEBUG=knex:query nodemon app.js",
    "db:create": "createdb --echo super_team_picker",
    "db:drop": "dropdb --echo --if-exists super_team_picker"
  },
```